src/relayserver/TransactionManager.ts

```diff
@@ -226,11 +226,12 @@ data         | 0x${transaction.data.toString('hex')}
       if (shouldRecheck) {
         const receipt = await this.contractInteractor.getTransaction(transaction.txId)
         if (receipt == null) {
-          log.warn(`failed to fetch receipt for tx ${transaction.txId}`)
+          log.warn(`warning: failed to fetch receipt for tx ${transaction.txId}`)
```

Comment:
That's actually fine, the transaction is not yet mined so the node does not give you a receipt.